yt/yt/tests/library/yt_queries.py

```diff
@@ -108,3 +108,7 @@ def abort_query(id, **kwargs):
 def alter_query(id, **kwargs):
     kwargs["query_id"] = id
     execute_command("alter_query", kwargs)
+
+def get_acos_for_queries(**kwargs):
+    return execute_command("get_acos_for_queries", kwargs, parse_yson=True)
+
```

Comment:
This is an extra empty line

Reply:
As intended, an empty line in the end of the file :)
I made a separate push for this change